io.dbeaver.DBeaverCommunity.Client.mariadb.metainfo.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>io.dbeaver.DBeaverCommunity.Client.mariadb</id>
+  <extends>io.dbeaver.DBeaverCommunity.desktop</extends>
```

Comment:
```suggestion
  <extends>io.dbeaver.DBeaverCommunity</extends>
```
Should be the id of the app you extend and not it's desktop-id

Reply:
My bad, I was using https://github.com/flathub/org.gajim.Gajim.Plugin.pgp/blob/master/org.gajim.Gajim.Plugin.pgp.metainfo.xml as a guide.

Fixed in both pulls.